### ISSUE TITLE
Initialization with StubBroker fails

### ISSUE THREAD

**Issue by Sovetnikov** [jmagnusson/sentry-dramatiq#7]
Seems there is error in handling middleware argument passed to StubBroker and Broker.

middlewares list passed to Broker as args[0] and then [args[0]] gives list(list) (i marked the line i'm taking about as ---> in code snippet below) resulting in this Exception:

>   File "venv\lib\site-packages\django_dramatiq\apps.py", line 107, in <module>
    DjangoDramatiqConfig.initialize()
  File "venv\lib\site-packages\django_dramatiq\apps.py", line 74, in initialize
    broker = broker_class(middleware=middleware, **broker_options)
  File "shared\dramatiq_utils.py", line 8, in __init__
    super(EagerBroker, self).__init__(*args, **kwargs)
  File "dramatiq\dramatiq\brokers\stub.py", line 33, in __init__
    super().__init__(middleware)
  File "venv\lib\site-packages\sentry_dramatiq\__init__.py", line 63, in sentry_patched_broker__init__
    original_broker__init__(self, *args, **kw)
  File "dramatiq\dramatiq\broker.py", line 84, in __init__
    self.add_middleware(m)
  File "dramatiq\dramatiq\broker.py", line 135, in add_middleware
    self.actor_options |= middleware.actor_options
AttributeError: 'list' object has no attribute 'actor_options'

```
    def sentry_patched_broker__init__(self, *args, **kw):
        hub = Hub.current
        integration = hub.get_integration(DramatiqIntegration)

        try:
            middleware = kw.pop('middleware')
        except KeyError:
            # Unfortunately Broker and StubBroker allows middleware to be
            # passed in as positional arguments, whilst RabbitmqBroker and
            # RedisBroker does not.
            if len(args) > 0:
                assert len(args) < 2
--->        middleware = None if args[0] is None else [args[0]]
                args = []
            else:
                middleware = None
```

**Comment by Sovetnikov** [comment]
There is pull request for this fix already :)

**Comment by ramonsaraiva** [comment]
The fix is now live on 0.3.2 https://pypi.org/project/sentry-dramatiq/0.3.2/
